src/Mapper/ProgrammesDbToDomain/MasterBrandMapper.php

```diff
@@ -91,6 +91,12 @@ private function getCompetitionWarningModel(array $dbMasterBrand, string $key =
             return null;
         }
 
+        if (array_key_exists('programmeItem', $dbMasterBrand[$key]) && is_null($dbMasterBrand[$key]['programmeItem'])) {
```

Comment:
So, it's fine if the programmeItem key doesn't exist?

Reply:
This is basically the difference between "Not queried" and "Not found". Hence using array_key_exists instead of isset